### PR TITLE
Fix Fly Speed

### PR DESCRIPTION
**Pull request by Maxetto**
According to [NoCheatPlus](https://github.com/NoCheatPlus/NoCheatPlus/blob/master/NCPCore/src/main/java/fr/neatmonster/nocheatplus/checks/moving/magic/Magic.java#L30) (and even some of your [datas](https://github.com/Xephi/AuthMeReloaded/blob/master/src/test/resources/fr/xephi/authme/data/backup/sample-folder/data.json#L13)) and some tests, default Fly speed is 0.1f.

### CODE REVIEW

**Comment by ljacqu** [comment]
Could you extract this value into a constant please?

**Comment by Maxetto** [comment]
Done (I think)... Do you want it for tests too?

**Comment by ljacqu** [comment]
Thanks for your work. I suggested a constant to have the value at one place as public, then it could be used everywhere :) 
I'm on mobile so can't check this float/double business.

**Comment by ljacqu** [comment]
Hi @Maxetto, thanks for this PR. I close it because I merged it manually to fix the compilation issues. 

The issue was that the following doesn't work in Java:
```java
public static final float DEFAULT_WALKSPEED = 0.1;
```

The field is declared as a `float` but decimals are `double`s by default. `double` has a more extended range of values than `float`, so it cannot implicitly convert from `double` to `float`. The solution is to add an `f` at the end to tell Java that we want a float:

```java
public static final float DEFAULT_WALKSPEED = 0.1f;
```

In other words:
```java
double d = 0.2; // works!
float f = 0.2f; // works! -> f suffix

double z = f; // works! float to double can be converted implicitly
float y = d; // does not work! needs to be explicit --> float y = (float) d; 
```
